package.json

```diff
@@ -52,7 +52,7 @@
     "bs-custom-file-input": "^1.3.4",
     "fuse.js": "^6.4.6",
     "jquery": "^3.6.0",
-    "jquery-autocomplete": "^1.2.8",
+    "jQuery-Autocomplete": "https://github.com/devbridge/jQuery-Autocomplete.git#v1.4.11",
```

Comment:
That caused an error in GitHub actions.

Reply:
The one you used was a completely different one. I will have a look into it again ;)

Reply:
So I added `npm run build` for the MergeRequests as well and it builds

Reply:
ahh the difference in the broken build was the protocol: https://github.com/RailwayStations/RSWebsite/runs/2555472899?check_suite_focus=true#step:6:5
The container of the action has no ssh key so he can not clone, with http it works

Reply:
That expains it and that is where my struggle started. Thanks a lot.